src/Game/AccountManagement.java

```diff
@@ -119,6 +105,40 @@ public static void register(String user, String pass) {
 		}
 	}
 	
+	public static void enc_cred_put(Object buffer, String str, int sessionId) {
+	    byte[] data = str.getBytes();
+	    int len = data.length;
+	    byte[] block = new byte[15];
+
+	    for (int i = 0; i < len; i += 7) {
+	      block[0] = (byte) ((int) (1.0D + Math.random() * 127.0D));
+	      block[1] = (byte) ((int) (Math.random() * 256.0D));
+	      block[2] = (byte) ((int) (Math.random() * 256.0D));
+	      block[3] = (byte) ((int) (Math.random() * 256.0D));
+
+
+	      StreamUtil.putIntTo(buffer, sessionId); // n = RSC127 session ID

```

Comment:
replace comment on this line, no longer makes sense
` // "session ID" is an RSC127 concept`

Reply:
placed